src/main/java/de/picturesafe/search/elasticsearch/config/FieldConfiguration.java

```diff
@@ -83,4 +84,19 @@
                 .put("additionalParameters", getAdditionalParameters())
                 .build();
     }
+
+    default boolean equalsBesidesName(FieldConfiguration conf) {
+        return new EqualsBuilder()
+                .append(getElasticsearchType(), conf.getElasticsearchType())
+                .append(isSortable(), conf.isSortable())
+                .append(isAggregatable(), conf.isAggregatable())
+                .append(isMultilingual(), conf.isMultilingual())
+                .append(getAnalyzer(), conf.getAnalyzer())
+                .append(isWithoutIndexing(), conf.isWithoutIndexing())
+                .append(getCopyToFields(), conf.getCopyToFields())
+                .append(getNestedFields(), conf.getNestedFields())
+                .append(getAdditionalParameters(), conf.getAdditionalParameters())
+                .append(getParent(), conf.getParent())
+                .isEquals();
```

Comment:
I left these methods out because they are just convenience methods for copyToFields and nestedFields. But as someone could possibly implement the interface in a different way, i will add them.